src/components/pager/pager.component.js

```diff
@@ -104,12 +104,18 @@ const Pager = ({
 
   const sizeSelector = () => {
     return (
-      <StyledDropdown
-        options={ Immutable.fromJS(pageSizeSelectionOptions) }
+      <StyledSelect
         value={ currentPageSize }
         onChange={ (ev) => { handleOnPagination(ev); } }
         data-element='page-select'
-      />
+      >
+        { pageSizeSelectionOptions.map(sizeOptions => (
```

Comment:
Very minor but before you merge you might want to change `sizeOptions` to `sizeOption` here as it's just refering to a single element in the array

Reply:
Changes have been made in the latest commit. Cheers for pointing this out. I agree it's minor but still something that should be changed for readability purposes 👍